docs/Editing/Auto-Suggest.md

```diff
@@ -182,11 +182,17 @@ Our second task looks like this. Note that the due date is shown in the Descript
 
 ![auto-suggest-edit-incorrect-task](../images/auto-suggest-edit-incorrect-task.png)
 
-### How do I see more suggestions?
+### How do I see less/more suggestions?
 
-Increase the 'Maximum number of auto-suggestions to show' value in settings (and re-start Obsidian) so that the menu will contain more options.
+In the Obsidian Settings, Navigate too Community Plugins.
+There find the Tasks Plugin, and go too Options via the Cog.
```

Comment:
None of the other Tasks documentation pages tell people how to find plugin settings, so I don't thing it needs to be said here.

But for future reference, these usages should be `to`, not `too` - see:

https://www.grammarly.com/blog/to-too/#:~:text=To%20is%20a%20preposition%20with,one%20of%20the%20other%20words.

> - To is a preposition with several meanings, including “toward” and “until.”
> - Too is an adverb that can mean “excessively” or “also.”

Reply:
> None of the other Tasks documentation pages tell people how to find plugin settings, so I don't think it needs to be said here.

Fixed